### DISCUSSION TITLE
generate automatic collection for website for each locale

### DISCUSSION THREAD

**Discussion by Dexus** [11ty/eleventy#4137] [Enhancement Queue]
### Is your feature request related to a problem? Please describe.

I need to create for each locale an own collection, but it would be cool, to have for each collection, that is automatic generated or created, it would also create a seperate one for each locale.

### Describe the solution you'd like

automatic create a collection that is defined for each locale

**Comment by pdehaan** [comment]
I think it mostly depends on how your locales/pages are set up.
If its "About" pages, you could always just do something like `tags: ["fr"]` in your front matter to add localized pages to auto add them to a locale collection.
Or if you have a lot of pages nested in a directory, you could do something like a directory data file, like "/it/it.11tydata.json" which adds all nested pages to an "it" collection:
```json
{
  "tags": ["it"]
}
```

Or, depending on how your site is organized, you could always do something like `.addCollection("de", collectionApi => {...})` in your eleventy.config.js file to manually create the locale collections based on some other criteria.

**Comment by Dexus** [comment]
I simply use the 2.0.1 lang tag in the parent folder to set the locale. But then the stipid is also that the categories/tags are not in the lists...

in the past wit 1.x i had created a list of locals that are supported, created each with `.addCollection` but now with the local support to create mulit-linguale websites, I think it would nice to have this created for the locals also automated.

**Comment by zachleat** [comment]
Lots of overlap with #4116

**Comment by zachleat** [comment]
Cross-link to #4051
